src/output/default.rs

```diff
@@ -53,7 +53,8 @@ fn wrap_with_comment(s: &str, c: &str, indent: &str) -> String {
         .map(|x| format!("{}{}", indent, &x.trim()))
         .collect();
     // Fit the comment onto the last line
-    let comment_space = tw - c.chars().count() - 2;
+    let c_count = c.chars().count();
+    let comment_space = if c_count + 2 < tw { tw - c_count - 2 } else { 0 };
```

Comment:
Use `std::cmp::max`, it will read more clearly (ie, `max(0, ...)` will ensure a minimum of zero

Reply:
The error is that the subtraction overflows, i.e. it becomes negative for an unsigned type. Using `std::cmp::max` won't stop that from happening. Intuitively, 0 is the lowest unsigned value, so doing a max with it and something else is simply the identity function.

Reply:
Aha, I had thought that it was `isize` (and I don't know why I thought that. 😄 ). Can we convert it to isize and then do the operation I propose? It should be much more resilient to changes over time elsewhere in the code, because it was already magic numbery due to my laziness, so I'd like to limit that proliferating haha.

Reply:
Personally, I prefer this approach. I agree that there are already magic numbers, etc. My opinion is that using `isize` and `max` would be more confusing.

In the end, the current code panics in debug builds and resolving that is more important than how. I'll let you decide which approach you think is best.

Reply:
How about just using `checked_sub` and `unwrap_or` on the result? In this case, `saturating_sub` might even do the trick.

Reply:
Aha, yes! I always forget about the wrapping-safe functions on the number types. :smile: